FPIS/Views/AddDesignation.Designer.cs

```diff
@@ -0,0 +1,144 @@
+            // materialTextBoxDesignationName
+            // 
+            this.materialTextBoxDesignationName.AnimateReadOnly = false;
+            this.materialTextBoxDesignationName.BorderStyle = System.Windows.Forms.BorderStyle.None;
+            this.materialTextBoxDesignationName.Depth = 0;
+            this.materialTextBoxDesignationName.Font = new System.Drawing.Font("Roboto", 16F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Pixel);
+            this.materialTextBoxDesignationName.Hint = "Designation";
+            this.materialTextBoxDesignationName.LeadingIcon = null;
+            this.materialTextBoxDesignationName.Location = new System.Drawing.Point(24, 191);
+            this.materialTextBoxDesignationName.MaxLength = 50;
+            this.materialTextBoxDesignationName.MouseState = MaterialSkin.MouseState.OUT;
+            this.materialTextBoxDesignationName.Multiline = false;
+            this.materialTextBoxDesignationName.Name = "materialTextBoxDesignationName";
+            this.materialTextBoxDesignationName.Size = new System.Drawing.Size(397, 50);
+            this.materialTextBoxDesignationName.TabIndex = 1;
+            this.materialTextBoxDesignationName.Text = "";
+            this.materialTextBoxDesignationName.TrailingIcon = null;
+            // 
+            // btnAddDesignation
+            // 
+            this.btnAddDesignation.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.btnAddDesignation.CharacterCasing = MaterialSkin.Controls.MaterialButton.CharacterCasingEnum.Normal;
+            this.btnAddDesignation.Density = MaterialSkin.Controls.MaterialButton.MaterialButtonDensity.Default;
+            this.btnAddDesignation.Depth = 0;
+            this.btnAddDesignation.HighEmphasis = true;
+            this.btnAddDesignation.Icon = null;
+            this.btnAddDesignation.Location = new System.Drawing.Point(131, 296);
+            this.btnAddDesignation.Margin = new System.Windows.Forms.Padding(4, 6, 4, 6);
+            this.btnAddDesignation.MouseState = MaterialSkin.MouseState.HOVER;
+            this.btnAddDesignation.Name = "btnAddDesignation";
+            this.btnAddDesignation.NoAccentTextColor = System.Drawing.Color.Empty;
+            this.btnAddDesignation.Size = new System.Drawing.Size(149, 36);
+            this.btnAddDesignation.TabIndex = 2;
+            this.btnAddDesignation.Text = "Add Designation";
+            this.btnAddDesignation.Type = MaterialSkin.Controls.MaterialButton.MaterialButtonType.Contained;
+            this.btnAddDesignation.UseAccentColor = false;
+            this.btnAddDesignation.UseVisualStyleBackColor = true;
+            this.btnAddDesignation.Click += new System.EventHandler(this.btnAddDesignation_Click);
+            // 
+            // labelDepartmentName
+            // 
+            this.labelDepartmentName.AutoSize = true;
+            this.labelDepartmentName.Location = new System.Drawing.Point(26, 150);
```

Comment:
Check the position for the label used in displaying the error message. Same issue I reported in the `AddDepartment` screen